### PR TITLE
Ek develop jan 2017 tests

### PR DESCRIPTION
**Pull request by ekaspern**
I need to update some of the widgets so the tests would not fail.

### CODE REVIEW

File: tests/components/progress_bar.coffee [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  ProgressBar = React.createFactory require('../../src/components/progress_bar')
+  TestUtils = require 'react-addons-test-utils'
+  ReactDOM = require 'react-dom'
+
+  # catchSpy = sinon.spy()
+
+  # window.PDFJS =
+  #   getDocument: (params) ->
+  #     then: ->
+  #       catch: catchSpy
```
Comment (darcyadams) [inline tests/components/progress_bar.coffee]: this can be removed

File: tests/components/pvr_info_list.coffee [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    }
+
+    height = height + 20
+
+    mainDiv = TestUtils.findRenderedDOMComponentWithClass pvrInfoList, 'plain-pvr'
+  
+    expect(mainDiv.style['height']).to.equal("#{height}px")
+
+
+  it 'Should set the div height based off of the height property and headerTitle property and items property is empy', ->
```
Comment (darcyadams) [inline tests/components/pvr_info_list.coffee]: typo... `property is empy`

File: tests/components/select_pvr_option.coffee [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+          disabled: yes
+      }
+    
+    mainDiv = TestUtils.findRenderedDOMComponentWithClass selectPvrOption, 'select-pvr-option'
+
+    TestUtils.Simulate.click mainDiv, {}
+
+    expect(handleClick.called).to.equal(false)
+
+  it 'Should NOT call props.handleClick when clicked if isSelected is true and canDeselect is false', ->
```
Comment (darcyadams) [inline tests/components/select_pvr_option.coffee]: what about?
`'Should call props.handleClick when clicked if isSelected is true and canDeselect is true`

Comment (ekaspern) [inline tests/components/select_pvr_option.coffee]: I have this one above... it is really 

'Should call the props.handleClick when clicked if disabled is false or (isSelected is false and canDeselect is true)'

**Pull request merged**